onboarding.md

```diff
@@ -0,0 +1,36 @@
+# Developers Onboarding
+
+## Welcome in to our developers hub 
+
+We look forward to working together!
+
+## Start Here 
+
+**What is Cambiatus?**
+
+Open online platform that enables communities/groups with environmental and social objectives to create and implement their own community currencies. 
+
+At Cambiatus we make the blockchain technology available and user friendly, so these communities can take full advantage of it and solve their environmental or social problems. Cambiatus is also becoming a tool to make DAC (decentralized autonomous communities) creation easy and fun. 
+
+**What is DAC?**
+
+We have put together a document called the [Path to Knowledge](https://docs.google.com/document/d/1cKmTSR9VpPkYyD-uhpnnAuF4JITGWSu6X5BP9le1yMw/edit) in which we offer some sources of information which you can follow to learn about all the above topics. 
+
+**Now what really means Cambiatus?**
+
+In Spanish “Cambia” is the word used to express “Change”.  Cambiatus refers to a change in our mindset, a change in the way we understand money, organizations and collaboration. 
+
+When people start using Cambiatus methodologies and tools, something changes inside them, they get it! 
+
+## Development Environments
+
+Now that you have your Cambiatus mindset properly setup is time to have some tech fun!
+
+Here are our development environments documentation to help you dive in and setup:
+
+- [Backend](backend.md)
+- Frontend
```

Comment:
we can add a link to the frontend here

Reply:
Can I suggest that we do the same way as the backend and create a documentation page for it?

Reply:
I'll go in pt-br here:

Acho que nao faz muito sentido a gente ter páginas dos projetos dentro da Wiki. Veja esse arquivo é sobre onboarding, faz parte do processo vc ir lá nos projetos e fazer seu setup. Na wiki podemos colocar outras coisas mais especificas que ficariam grande demais lá no README de cada projeto. Por exemplo, acho que colocar técnicas de debug lá no readme demais. Já na wiki essas coisas podem existir numa boa.

Reply:
Understood!!